README.md

```diff
@@ -269,3 +269,21 @@ Those rules can be written according to this template :
 ```
 
 [unit testing rules]: https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/
+
+## Alertmanager inhibition dependency check
+
+In order for Alertmanager inhibition to work we need 3 elements:
+  - an Alert rule with some source labels
+  - an Inhibition definition mapping source labels to target labels
+  - an Alert rule with some target labels
+
+This is what we call the inhibition dependency chain.
+
+One can check whether inhibition labels (mostly "cancel_if_" prefixed ones) are well defined and triggered by a corresponding label in the alerting rules by running the `make test-inhibitions` command at the projet's root directory.
+
+This command will output the list of missing labels. Each of them will need to be defined in the alerting rules.
+If there is no labels outputed, this means tests passed and did not find missing inhibition labels.
+
+Warning: the tool may output false alerts or miss some alerts because of the following limitations.
```

Comment:
I think the "limitations" part is outdated. @QuantumEnigmaa , maybe you want to change that part?

Reply:
I think the second assertion can be considered false. However I'm not sure about the 1st one : the script definitely check labels for each provider but I still have to check whether it throws errors if an inhibition is missing for a specific provider.

Reply:
merging this. feel free to update when needed.